pkg/apiserver/infrastructure/datastore/kubeapi/kubeapi.go

```diff
@@ -84,6 +84,9 @@ func (m *kubeapi) generateConfigMap(entity datastore.Entity) *corev1.ConfigMap {
 	if labels == nil {
 		labels = make(map[string]string)
 	}
+	for k, v := range labels {
```

Comment:
It would be better to add some test cases.

Reply:
i will add some test cases to test the verifyValue function's effectiveness